### ISSUE TITLE
Mod Interaction between Chisel & Bits

### ISSUE THREAD

**Issue by FallenEagle1** [yrsegal/PSIonic-Upgrades#15]
Version: r1.15
Psi Version: r1.0-42
Chisel & Bits Version: 12.11

Blocks in question: Colored Psimetal Blocks, both normal and dark

This is kind of two issues, but I'm thinking the two are related.

Problem 1: This was found in single player. When trying to chisel any of the colored psimetal blocks, you can get the bits, and they are named properly, but the texture defaults to the default psimetal block. Once the block becomes complete again, the correct texture is shown.

Problem 2: This was found in multiplayer, and I haven't be able to replicate in single. The set up is similar to problem one. However, when you click to get bits, the block in the world changes to default, looks like some bits have been pulled from it, but you do not get any bits in your inventory. Furthermore, if you update the block (right click, left click, etc) the block becomes whole again and has the correct texture.

Basically the the blocks cannot be broken down properly with chisel & bits. I'm thinking the issue is on this side, and not chisel & bits, because Vazkii had a similar problem with the default blocks. The issue where this was brought up was https://github.com/AlgorithmX2/Chisels-and-Bits/issues/49 

It was brought up there, and then went to Psi's git hub through a link posted there.
